applications/milliwatt/src/milliwatt_listener.erl

```diff
@@ -26,10 +26,7 @@
 -record(state, {}).
 -type state() :: #state{}.
 
--define(BINDINGS, [{'route', [{'types', ?RESOURCE_TYPES_HANDLED}
-                             ,{'restrict_to', ['no_account']}
```

Comment:
why are you removing this ? afaik milliwatt requests are supposed not to be autheticated